### PR TITLE
Make configured component branch available in scripts as WL_BRANCH

### PR DESCRIPTION
**Pull request by crisp2u**
Before submitting pull request, please ensure that:

- [x] Every commit has message which describes it
- [x] Every commit is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any code changes come with testcase
- [x] Any new functionality is covered by user documentation

### CODE REVIEW

**Comment by nijel** [comment]
Can you please add it to the documentation as well?

**Comment by crisp2u** [comment]
Done (sorry, I was not sure from the start were is the source for the docs)

File: weblate/trans/scripts.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def run_hook(component, translation, script, env=None, *args):
             'WL_PATH': target.get_path(),
             'WL_FILEMASK': component.filemask,
             'WL_TEMPLATE': component.template,
-            'WL_FILE_FORMAT': component.file_format,
```
Comment (nijel) [inline weblate/trans/scripts.py]: I guess that the removal is not intentional...

**Comment by nijel** [comment]
Thanks for the docs! However I've not noticed another problem with the patch before...

**Comment by nijel** [comment]
Merged with dd65d4702 to fix removed variable.

**Pull request merged**